core/GlobalState.cc

```diff
@@ -832,6 +845,9 @@ void GlobalState::initEmpty() {
     // Collect size prior to loop since singletons will cause vector to grow.
     size_t classAndModulesSize = classAndModules.size();
     for (uint32_t i = 1; i < classAndModulesSize; i++) {
+        if (!classAndModules[i].isClassModuleSet()) {
+            classAndModules[i].setIsModule(true);
```

Comment:
1. It seemed more verbose. Maybe that's fine.
2. We'd still have to loop over `classesAndModules` here anyways to ensure that one of `true` or `false` was set.
3. Either the value of `true` or `false` doesn't matter, or it will be reported as an error from the `sorbet-orig` bootstrapping run of Sorbet if there was some sort of mismatch, so there's low risk of the default `setIsModule(true)` being wrong (if it's wrong, it's going to get caught)